chroma_core/lib/util.py

```diff
@@ -163,22 +163,24 @@ def __str__(self):
 
 
 class CommandLine(object):
-    def try_shell(self, cmdline, mystdout = subprocess.PIPE,
-                  mystderr = subprocess.PIPE, stdin_text = None):
-        rc, out, err = self.shell(cmdline, mystdout, mystderr, stdin_text)
+    def try_shell(self, cmdline, mystdout=subprocess.PIPE,
+                  mystderr=subprocess.PIPE, stdin_text=None, shell=False):
+        rc, out, err = self.shell(
+            cmdline, mystdout, mystderr, stdin_text, shell=shell)
```

Comment:
Where is "shell" coming from here?

Reply:
https://docs.python.org/2/library/subprocess.html#popen-constructor